### PR TITLE
apply logging configurations

### PR DESCRIPTION
**Pull request by dangel101**
Currently the configurations set at logging_conf aren't being applied
Thus, the log file isn't being created as expected
This patch applies these configurations

@mnecas @mwperina 
@pcuzner @jmolmo 
please review

### CODE REVIEW

**Comment by dangel101** [comment]
@pcuzner fixed, thanks!

File: runner_service/configuration.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import yaml
 import getpass
 import logging
+import logging.config
```
Comment (jmolmo) [inline runner_service/configuration.py:6]: Not needed. Redundant

Comment (pcuzner) [reply runner_service/configuration.py:6]: ?
`Python 3.7.7 (default, Mar 13 2020, 10:23:39)
[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import logging
>>> dir(logging)
['BASIC_FORMAT', 'BufferingFormatter', 'CRITICAL', 'DEBUG', 'ERROR', 'FATAL', 'FileHandler', 'Filter', 'Filterer', 'Formatter', 'Handler', 'INFO', 'LogRecord', 'Logger', 'LoggerAdapter', 'Manager', 'NOTSET', 'NullHandler', 'PercentStyle', 'PlaceHolder', 'RootLogger', 'StrFormatStyle', 'StreamHandler', 'StringTemplateStyle', 'Template', 'WARN', 'WARNING', '_STYLES', '_StderrHandler', '__all__', '__author__', '__builtins__', '__cached__', '__date__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '__status__', '__version__', '_acquireLock', '_addHandlerRef', '_after_at_fork_child_reinit_locks', '_at_fork_reinit_lock_weakset', '_checkLevel', '_defaultFormatter', '_defaultLastResort', '_handlerList', '_handlers', '_levelToName', '_lock', '_logRecordFactory', '_loggerClass', '_nameToLevel', '_register_at_fork_reinit_lock', '_releaseLock', '_removeHandlerRef', '_showwarning', '_srcfile', '_startTime', '_warnings_showwarning', 'addLevelName', 'atexit', 'basicConfig', 'captureWarnings', 'collections', 'critical', 'currentframe', 'debug', 'disable', 'error', 'exception', 'fatal', 'getLevelName', 'getLogRecordFactory', 'getLogger', 'getLoggerClass', 'info', 'io', 'lastResort', 'log', 'logMultiprocessing', 'logProcesses', 'logThreads', 'makeLogRecord', 'os', 'raiseExceptions', 'root', 'setLogRecordFactory', 'setLoggerClass', 'shutdown', 'sys', 'threading', 'time', 'traceback', 'warn', 'warning', 'warnings', 'weakref']
>>> import logging.config
>>> dir(logging.config)
['BaseConfigurator', 'ConvertingDict', 'ConvertingList', 'ConvertingMixin', 'ConvertingTuple', 'DEFAULT_LOGGING_CONFIG_PORT', 'DictConfigurator', 'IDENTIFIER', 'RESET_ERROR', 'StreamRequestHandler', 'ThreadingTCPServer', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__spec__', '_clearExistingHandlers', '_create_formatters', '_handle_existing_loggers', '_install_handlers', '_install_loggers', '_listener', '_resolve', '_strip_spaces', 'dictConfig', 'dictConfigClass', 'errno', 'fileConfig', 'io', 'listen', 'logging', 're', 'stopListening', 'struct', 'sys', 'threading', 'traceback', 'valid_ident']`

logging.config is required

Comment (jmolmo) [reply runner_service/configuration.py:6]: @pcuzner: you're right!

**Pull request merged**